docs/DeveloperGuide.md

```diff
@@ -303,6 +307,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case ends.
 
 
+**Use case: UC03 - Edit a property**
+
+**MSS**
+
+1. User request to edit a property
+2. PocketEstate shows an updated list of properties
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. The property to be edited has unrecognized fields.
+
+    * 1a1. PocketEstate updates the recognized fields.
```

Comment:
Should this be left out because the use cases are supposed to show only what the user can see?

Reply:
i was thinking like the recognized fields will still be updated and shown to be updated in the list, should i phrase it differently?

Reply:
I guess this is down to the implementation already. 
Will you want to 
1. execute the edit command even if there are erroneous fields or 
2. abort the whole command entirely and show an error?

Reply:
personally, i think i would prefer 1 if i were a user but would also want to be alerted that there were erroneous fields however,  i think it would be best to follow the add command. Like if add aborts the whole command, i think edit should abort the whole command too just to be consistent

Reply:
The add command follows number 2 for now.
Good point raised, we can follow up on this discussion about which format we want with the other team members.